vllm/model_executor/models/mamba.py

```diff
@@ -332,40 +282,20 @@ def forward(
             current_ssm_state = ssm_state[i]
             current_conv_state = conv_state[i]
 
-            hidden_states, residual = layer(
+            hidden_states = layer(
                 positions=positions,
                 hidden_states=hidden_states,
                 attn_metadata=attn_metadata,
                 residual=residual,
```

Comment:
Residual is still passed in here. I personally think it is fine to keep the previous structure of passing residual into rmsnorm, but up to you